modules/backup-gcs/client.go

```diff
@@ -122,7 +122,7 @@ func (g *gcsClient) AllBackups(ctx context.Context) ([]*backup.DistributedBackup
 		return nil, nil
 	}
 
-	iter := bucket.Objects(ctx, nil)
+	iter := bucket.Objects(ctx, &storage.Query{Prefix: g.config.BackupPath, MatchGlob: "*json"})
```

Comment:
**question (non-blocking):** is matchGlob intended ?

Reply:
yes?

Reply:
we're looking for the backup status files

Reply:
i am just asking because the pattern wasn't in the other modules

Reply:
The other modules don't have globs, that I can find

Reply:
maybe we should also write the status files to a single directory?

Reply:
you'r call, if it's not much work. i think the `*.config.BackupPath` would be already fine to limit the query and sufficient
 
if changing to custom static dir. under the configured path for status files would be much work, i'd say let's create a ticket to backlog it and we can revisit it later if needed as enhancement